tests/acceptance/features/apiSpaces/setQuota.feature

```diff
@@ -0,0 +1,178 @@
+@api
+Feature: Set quota
+  As a user
+  I want to set quota to different users
+  So that users can only take up a certain amount of storage space
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+
+
+  Scenario Outline: admin sets personal space quota of user with different role
+    Given the administrator has given "Alice" the role "<role>" using the settings api
+    And the administrator has given "Brian" the role "<userRole>" using the settings api
+    When user "Alice" changes the quota of the "Brian Murphy" space to "100" owned by user "Brian"
+    Then the HTTP status code should be "200"
+    And for user "Brian" the JSON response should contain space called "Brian Murphy" and match
+    """
+    {
+      "type": "object",
+      "required": [
+        "quota"
+      ],
+      "properties": {
+        "quota": {
+          "type": "object",
+          "required": [
+            "total"
+          ],
+          "properties": {
+            "total" : {
+              "type": "number",
+              "enum": [100]
+            }
+          }
+        }
+      }
+    }
+    """
+    Examples:
+      | role  | userRole    |
+      | Admin | Admin       |
+      | Admin | Space Admin |
+      | Admin | User        |
+      | Admin | Guest       |
+
+
+  Scenario Outline: user other than admin can't set the personal space quota of other users
+    Given the administrator has given "Alice" the role "<role>" using the settings api
+    And the administrator has given "Brian" the role "<userRole>" using the settings api
+    When user "Alice" changes the quota of the "Brian Murphy" space to "100" owned by user "Brian"
+    Then the HTTP status code should be "401"
+    And for user "Brian" the JSON response should contain space called "Brian Murphy" and match
+    """
+    {
+      "type": "object",
+      "required": [
+        "quota"
+      ],
+      "properties": {
+        "quota": {
+          "type": "object",
+          "required": [
+            "total"
+          ],
+          "properties": {
+            "total" : {
+              "type": "number",
+              "enum": [0]
+            }
+          }
+        }
+      }
+    }
+    """
+    Examples:
+      | role        | userRole    |
+      | Space Admin | Admin       |
+      | Space Admin | Space Admin |
+      | Space Admin | User        |
+      | Space Admin | Guest       |
+      | User        | Admin       |
+      | User        | Space Admin |
+      | User        | User        |
+      | User        | Guest       |
+      | Guest       | Admin       |
+      | Guest       | Space Admin |
+      | Guest       | User        |
+      | Guest       | Guest       |
+
+
+  Scenario Outline: admin or space admin user sets quota of a space
```

Comment:
```suggestion
  Scenario Outline: admin or space admin user sets a quota of a project space
```
Seems easier to figure out whether a scenario is about personal or project space.